PhoneBook-App/Controllers/PhoneBookController.cs

```diff
@@ -0,0 +1,75 @@
+﻿
+internal class PhoneBookController
+{
+    readonly Database Database = new();
+    readonly TableVisualisationEngine TVEngine = new();
+    internal void ShowMenu()
+    {
+        Console.Clear();
+        Console.WriteLine("1. Show contacts");
+        Console.WriteLine("2. Add contact");
+        Console.WriteLine("3. Delete contact");
+        Console.WriteLine("4. Update contact");
+        Console.WriteLine("5. Exit");
+    }
+
+    internal void Navigate(int Option)
+    {
+        switch (Option)
+        {
+            case 1:
+                ShowContacts();
+                break;
+            case 2:
+                AddContact();
+                break;
+            case 3:
+                DeleteContact();
+                break;
+            case 4:
+                UpdateContact();    
+                break;
+            case 5:
+                Environment.Exit(0);
+                break;
+            default:
+                Console.WriteLine("Wrong Option!");
+                break;
+        }
+    }
+
+    private void ShowContacts()
```

Comment:
![image](https://user-images.githubusercontent.com/108711476/202897785-3043d690-dc93-480f-ad2c-9beadea6eacf.png)
This is a personal opinion but I would refrain from using abbreviations when naming things. You want code to be easily readable. Figuring out what "TV" means will consume precious seconds of someone's life 😁